Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
The additional check doesn't provide any additional safeguard (module A's unit tests should already guarantee functional correctness), but would necessitate an update to module A, even if the module C update doesn't create any functional regression, and even if no code change was made to module A, simply because an update to module B causes a new module C to be pulled in and cause dependency graph error in module A.

TL;DR: Explicitly specifying module C in module A's `Package.swift` can cause breakage in module A even if no change was made to module A and no breaking change was introduced in module C.